### PR TITLE
Updated React and React Native links

### PR DESCRIPTION
**Pull request by visshaljagtap**

### CODE REVIEW

File: content/blog/2015-05-01-graphql-introduction.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "GraphQL Introduction"
 author: [schrockn]
 ---
 
-At the React.js conference in late January 2015, we revealed our next major technology in the React family: [Relay](http://facebook.github.io/react/blog/2015/02/20/introducing-relay-and-graphql.html). 
+At the React.js conference in late January 2015, we revealed our next major technology in the React family: [Relay](https://reactjs.org/blog/2015/02/20/introducing-relay-and-graphql.html).
```
Comment (threepointone) [inline content/blog/2015-05-01-graphql-introduction.md]: This should be a relative link instead

Comment (visshaljagtap) [reply content/blog/2015-05-01-graphql-introduction.md]: Thanks for pointing out!

Comment (visshaljagtap) [reply content/blog/2015-05-01-graphql-introduction.md]: Updated the URL which is having relative link

**Comment by threepointone** [comment]
Thanks very much for this!

**Pull request merged**